src/css/custom.css

```diff
@@ -494,6 +495,26 @@ div[class^='announcementBar_'] {
   background-color: rgba(255, 32, 32, 20%);
 }
 
+.language-table td.fill {
+  padding: 0;
+  text-align: left;
+}
+
+.font-color {
+  background: linear-gradient(90deg, var(--color1), var(--color2));
+  position: relative;
+}
+.font-color::before {
```

Comment:
these should probably be scoped to `td` elements, also `.language-table` doesn't feel like a good name anymore

Reply:
Yeah it's definitely more flexible than that, but I'm not sure what to name it.